README.md

```diff
@@ -1,3 +1,104 @@
+
+4. Once the containers are built and started, you can go inside the app container: `docker exec -it devcontainer-hof-ecs-app-1 sh` (note: Docker containers may be named differently)
+
+5. Run the necessary commands to install dependencies `yarn` and `yarn start:dev` to start your application.
+
+## Install & Run the Application locally with VS Code Dev Containers
+
+Alternatively, if you are using [Visual Studio Code](https://code.visualstudio.com/) (VS Code), you can run the application with a [VS Code Dev Containers](https://code.visualstudio.com/docs/devcontainers/containers).
+
+The `.devcontainer` folder contains the necessary configuration files for the devcontainer.
+
+### Prerequisites
+   - [Docker](https://www.docker.com)
+   - [VS Code Dev Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extention
+
+### Setup
+
+By following these steps, you should be able to run your application using a devcontainer in VS Code. The Dev Containers extension lets you use a Docker container as a full-featured development environment. This provides a consistent development environment across different machines and ensures that all required dependencies are available. A `devcontainer.json` file in this project tells VS Code how to access (or create) a development container with a well-defined tool and runtime stack.
+
+1. Make sure you have Docker installed and running on your machine. Docker is needed to create and manage your containers.
+
+2. Install the [Dev Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extention in VS Code. This extension allows you to develop inside a containerised environment.
+
+3. To configure your dev environment, copy `/.devcontainer/devcontainer.env.sample` to `devcontainer.env` in the same directory and fill in the necessary values. This ensures your development container is set up with the required environment variables.
+
+4. Run the `Dev Containers: Open Folder in Container...` command from the Command Palette (F1) or click on the Remote Indicator (≶) in the status bar. This command will build and start the devcontainer based on the configuration files in the `.devcontainer` folder.
+
+7. Once the devcontainer is built and started, you will be inside the containerised environment. You can now work on your project as if you were working locally, but with all the necessary dependencies and tools installed within the container.
+
+8. To start the application, open a terminal within VS Code by going to `View -> Terminal` or by pressing `Ctrl+backtick` (`Cmd+backtick` on macOS). In the terminal, navigate to the project directory if you're not already there.
+
+9. Run the necessary commands to install dependencies `yarn` and `yarn start:dev` to start your application.
+
+### Testing
+
+#### Linting Tests
+`$ yarn run test:lint`
```

Comment:
can just be `yarn test:lint` , assumed that users are familiar with Yarn shorthand commands